OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -55,6 +55,7 @@ public enum ReadyTextStyleOptions { Solid, AlternatingColor, Blinking }
 		// Note: LinterHotkeyNames assumes that these are disabled by default
 		public readonly string HotkeyPrefix = null;
 		public readonly int HotkeyCount = 0;
+		public readonly HotkeyReference SelectProductionBuildingHotkey = new HotkeyReference();
```

Comment:
I just noticed that this has been added to the `ProductionPaletteWidget` rather than in a new `SelectProductionActorsHotkeyLogic` class. Was there a technical reason to do it this way rather than copying and adjusting `CycleProductionActorsHotkeyLogic`? If not, can we please change this?

Reply:
I added it to the widget because that was the easiest way to get the currently selected queue. How should I go about to obtain that information outside of the widget?

Reply:
Good point. Lets stick with what you have here then.